source/openapi3/split.yaml

```diff
@@ -4736,7 +4646,7 @@ components:
         description: The SuperPackage
         matures_at: '2021-06-13T00:00:00Z'
         your_bank_account_id: 83623359-e86e-440c-9780-432a3bc3626f
-        channels: ["new_payments_platform"]
+        channels: ["direct_entry"]
```

Comment:
FYI we may be removing channels altogether, not sure if the response still includes this.

For ref: https://github.com/zeptofs/nz-api-documentation/pull/11

Reply:
Nice 👌 
Seeing that is merged I'll pluck this out of the response 👍